modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,292 @@
+[[retail-server-setup]]
+= Retail Uyuni Server Setup
+
+This section covers {productname} Server setup, using these procedures:
+
+* Start {productname} setup with {yast}
+* Create the main administration account with the {productname} {webui}
+* Name your base organization and add login credentials
+* Synchronize the openSUSE Leap channel
++
+// ^^^ CHECKIT
+
+
+
+[[retail-server-setup-yast]]
+== Set up {productname} with {yast}
+
+This section guides you through {productname} setup procedures.
+
+.Procedure: {productname} Setup
+. On the {productname} Server, at the command prompt, as root, start {yast}:
++
+----
+yast2
+----
+
+. Navigate to menu:Network Services[Uyuni Setup] to begin set up.
+
+. From the introduction screen, select menu:Uyuni Setup[Set up Uyuni from scratch] and click btn:[Next] to continue.
+
+. Type an email address to receive status notifications and click btn:[Next] to continue.
+{productname} can sometimes send a large volume of notification emails.
```

Comment:
These will need the indent. Two spaces is sufficient, which will line it up with the previous line (possibly less ugly than a tab/four spaces?).